config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "playerBrandingId" : "e48c9b51282d406f957c666149763424",
```

Comment:
Is this an actual playerBrandingId? Do we want to include an actual playerBrandingId as default here?